Saudox/resources/views/welcome.blade.php

```diff
@@ -3,14 +3,88 @@
 
     <div class="desktop">
         <div class="infos-clinica">
-            <div class="imagens-slide">
-                <img class="slides" src="https://avatars3.githubusercontent.com/u/64805526?s=400&u=e4188ff9af3c0927962a181f241fbee79c506f4d&v=4">
-            </div>
-            <div class="info-slide-atual">
-                <h3 class="title-part">Sobre o slide atual:</h3>
-                <h4> Info 1: Sobre a empresa (objetivo geral)</h4>
-                <h4> Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor</h4>
-                <h4> Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor</h4>
+            <div id="carousel_indicadores" class="carousel slide" data-ride="carousel">
```

Comment:
No caso, isso é do bootstrap? Ou vc colocou isso no arquivo do CSS? Pq se for a segunda alternativa, acho que vc esqueceu de incluir o arquivo de css